src/sql/workbench/contrib/assessment/common/consts.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export enum AssessmentTargetType {
+	Server = 1,
+	Database = 2
+}
+
+export enum AssessmentType {
+	AvailableRules = 1,
+	InvokeAssessment = 2
+}
+
+export const TARGET_ICON_CLASS: { [targetType: number]: string } = {
+	[AssessmentTargetType.Database]: 'database',
+	[AssessmentTargetType.Server]: 'server-page'
+};
+
+
+export const TELEMETRY_VIEW = 'AssessmentTab';
```

Comment:
This should go in https://github.com/microsoft/azuredatastudio/blob/master/src/sql/platform/telemetry/common/telemetryKeys.ts#L63